### ISSUE TITLE
Document how to upgrade an existing deployment on 5.x to 6.x

### ISSUE THREAD

**Issue by Mpdreamz** [elastic/azure-marketplace#152]
Once we support `6.0` as per #151

**Comment by quimrodri** [comment]
Any update about this? Or link? Is it still planned to do? Thanks!

**Comment by russcam** [comment]
@quimrodri yes, there is still a plan to provide some guidance for this. I can't provide a timescale yet for when this will be. In the meantime, take a look at the general [Upgrade Elasticsearch guidance](https://www.elastic.co/guide/en/elasticsearch/reference/6.3/setup-upgrade.html) as it is equally applicable to running a cluster on Azure.

**Comment by mayuvara** [comment]
Do we have an update on this? Thanks!

**Comment by russcam** [comment]
Updated the title to updating from 6.x to 7.x.

The general [Upgrade Elasticsearch guidance](https://www.elastic.co/guide/en/elasticsearch/reference/6.3/setup-upgrade.html) is equally applicable to running a cluster on Azure.
